.eslintrc.js

```diff
@@ -0,0 +1,16 @@
+module.exports = {
```

Comment:
What is this file for?

Reply:
Eslint configuration. I don't have it setup globally